middleware.go

```diff
@@ -23,7 +23,7 @@ func DeviceMetadataMiddleware(delegate http.Handler) http.Handler {
 			metadata.SetSessionID(ksuid.New().String())
 
 			if auth, ok := bascule.FromContext(ctx); ok {
-				metadata.SetClaims(auth.Token.Attributes().FullView())
+				metadata.SetClaims(auth.Token.Attributes().(rawAttributes).GetRawAttributes())
```

Comment:
We should add a check to see if it is a `RawAttributes`

```suggestion
				if tokenAttributes, ok := auth.Token.Attributes().(RawAttributes); ok {
					metadata.SetClaims(tokenAttributes.GetRawAttributes())
				}
```